src/PageManagerServiceProvider.php

```diff
@@ -41,6 +41,14 @@ public function boot()
         $this->publishes([__DIR__.'/resources/lang' => resource_path('lang/vendor/backpack')], 'lang');
 
         $this->mergeConfigFrom(__DIR__.'/config/pagemanager.php', 'backpack.pagemanager');
+
+        // first load published/overwritten views
+        $customViewsFolder = resource_path('views/vendor/backpack/crud');
+        if (file_exists($customViewsFolder)) {
+            $this->loadViewsFrom($customViewsFolder, 'pagemanager');
+        }
```

Comment:
Great catch! Thanks for submitting the PR @mahasadhu , and for reviewing @promatik .

I have only one worry regarding this particular implementation - I believe there's a side-effect to it. If we do this, I think that EVERYTHING inside the `resources/views/vendor/backpack/crud` folder will then be available both inside the `backpack` and the `pagemanager` view namespace.

This doesn't sound like a big problem, I know, but... it's a bit bloaty and unintuitive to have `pagemanager::fields.table` also available, even if this package does NOT provide that field at all. In a sense, the package would take over something that it doesn't belong to it.

Hmm... How about this... What if inside the CrudController we change the `view_namespace` in case the view was published to the regular fields directory? I think something like this should work just as well, but without the downside above:

```diff
$this->crud->addField([
    'name' => 'template',
    'type' => 'select_page_template',
-    'view_namespace'  => 'pagemanager::fields',
+    // load the field from the normal fields folder if it was published there
+    'view_namespace'  => file_exists(resource_path('views/vendor/backpack/crud/fields/select_page_template.blade.php')) ? null : 'pagemanager::fields',
```

What do you guys think?

Reply:
I hadn't thought on it that way, indeed it's much better 👌
Committed and tested with your code, and it's working great.

Reply:
@tabacitu ah yes you're right, hadn't think of it that way...

thank you for checking!